### PR TITLE
pyro: rearrange pokes; allow FE pokes

### PR DESCRIPTION
**Pull request by hosted-fornet**
**Problem**:

Frontend cannot interact with %pyro.

**Solution**:

Rearrange %pyro and add `+enjs` and `+dejs` cores and appropriate `mar` files.

**Notes**:

~~WIP~~

I'll do a similar unification for %ziggurat scries/sub updates in #355 to reduce merge headaches.

### CODE REVIEW

**Comment by hosted-fornet** [comment]
FYI @0x70b1a5

File: app/pyro.hoon [hunk 1]
```diff
@@ -721,14 +643,106 @@
   ::
       %clear-snaps
     `state(fleet-snaps ~)
+  ::
+      %pill
+    (poke-pill pill.act)
```
Comment (tadad) [inline app/pyro.hoon:648]: I think it's cleaner to just in line +poke-pill here

**Review by tadad — COMMENTED** [review]
small style comment on poke-pill, otherwise looking good

File: app/pyro.hoon (hunk 1 above)
Comment (hosted-fornet) [reply app/pyro.hoon]: Good call, inlined in 2980c95

**Pull request merged**